### PR TITLE
🐛 replaced mentions to ampstart.com

### LINKED ISSUE #2162: Remove mentions of ampstart.com
Some of the documents on amp.dev still mention ampstart.com as a learning resource to users. Documents that do so need to be rephrased.

### PR DESCRIPTION
**Pull request by dlemm**
fix #2162 replace links to ampstart.com with a link to amp.dev/../templates/.. 
however there are still mentions in files (See image below). The .html-files contain only css-classes named "ampstart"

ampstart

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/responsive_design.md [hunk 1]
```diff
@@ -184,5 +184,5 @@ Here are some examples that we hope inspire you to create responsive AMP pages:
 #### Made by AMP
 
 - [Examples]({{g.doc('/content/amp-dev/documentation/examples/index.html', locale=doc.locale).url.path}})
-- [AMP Start templates](https://www.ampstart.com/)
+- [AMP Start templates]({{g.doc('/content/amp-dev/documentation/templates/index.html', locale=doc.locale).url.path}})
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/responsive_design.md]: Rephrase link text to just "Templates" - also in the i18n versions.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/optimize_amp.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ For example, by applying the following optimization techniques:
 *   [Preloaded hero image](#preload-hero-images) (the image size/encoding itself has not been changed)
 *   [Optimizes custom fonts](#optimize-custom-fonts) (in this case, Google fonts)
 
-the ["The Scenic" AMPStart template](https://www.ampstart.com/templates) loads [two seconds faster on a 3G connection](https://www.webpagetest.org/video/compare.php?tests=180529_RY_9198dcdba1824c169887c6e40c221dae-r:1-c:0).
+the ["The Scenic" AMPStart template]({{g.doc('/content/amp-dev/documentation/templates/index.html', locale=doc.locale).url.path}}) loads [two seconds faster on a 3G connection](https://www.webpagetest.org/video/compare.php?tests=180529_RY_9198dcdba1824c169887c6e40c221dae-r:1-c:0).
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/optimize_amp.md]: Rephrase to just ""The Scenic" template".

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/responsive_design.md (hunk 1 above)
Comment (dlemm) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/responsive_design.md]: Done

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/optimize_amp.md (hunk 2 above)
Comment (dlemm) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/optimize_amp.md]: Done

**Pull request merged**